specs/mediawiki/v1/content.yaml

```diff
@@ -788,6 +849,8 @@ paths:
             $ref: '#/definitions/problem'
       x-backend-request:
         uri: /{domain}/sys/parsoid/transform/html/to/wikitext{/title}{/revision}
+        body:
+          html: '{html}'
```

Comment:
We need to whitelist `oldid` and `scrubWikitext` as well, since these are sent by VE, cf. https://github.com/wikimedia/mediawiki-extensions-VisualEditor/blob/master/ApiVisualEditor.php#L158-L160